### ISSUE TITLE
Traits Missing Details

### ISSUE THREAD

**Issue by bpleshek** [ekolis/FrEee#45]
**Describe the bug**
The traits tab of the Empire creation screen has just a point value((1000 pts, 2000 pts, etc)) without any description of what you get for said points.

**To Reproduce**
1. Create a new race.
2. Go to the Traits Tab
3. Look at what isn't there.

**Expected behavior**
I expect it to say something like:
Brilliant Scientists (1000 pts) +10 science

or something like that.

**Actual behavior**
It only says:
(1000 pts)
(1000 pts)
(1000 pts)
(1500 pts)
(2000 pts)

**Screenshots**
![image](https://user-images.githubusercontent.com/12614696/71744508-7936cb80-2e35-11ea-954b-5b20ce26beef.png)

**Additional context**
N/A

**Comment by ekolis** [comment]
Hmm, I can't reproduce this bug - the text is visible for me. Could be an issue with the default foreground color being different on our PCs - what version of Windows and .NET are you running?

**Comment by bpleshek** [comment]
Visual Studio Enterprise Edition 2017(v15.9.18).  The Miscrosoft .NET framework version is 4.8.03752, and the project is set to 4.8 in the project properties.

I also have Visual Studio Enterprise 2019.  I could try that if you think it'd make a difference.

**Comment by bpleshek** [comment]
(deleted lots of paragraphs of my comments as I was stepping through the code).

I noticed that the tooltip still worked, even though it shows what I have in the picture above.  So I resized the form and the data repainted properly increasing the size of the checkboxes too.  

I have a video here for you to view showing the error and how resizing "fixes" it:

https://ttprivatenew.s3.amazonaws.com/pulse/bpleshek/attachments/12261060/FrEeeIssue.mp4

You can see that I attempt to click on the checkbox and it works over the (1000 pts) but not over the blank space.  Then after the resize, it works all the way out onto the description.

**Comment by ekolis** [comment]
Huh, that's interesting; I wonder why resizing the window would fix it? I don't think I'm doing any custom painting logic on this form.

By the way, how did you get the game to load so quickly? Did you time compress your video? The mod takes me longer than that to load, even though I do have a SSD...

**Comment by bpleshek** [comment]
I did not compress the video.  I don't think I have an SSD, but I'm not 100% sure.  It's a work machine and it always seemed slow to me, so I just assumed it wasn't.  Does your setup do a rebuild before every run even if nothing has changed?  By the way, I probably live less than 30 minutes from you.  Go figure out of everyone on the internet.....

**Comment by ekolis** [comment]
Rebuild before run? No, I'm saying once the game is running, when you load it up, the mod files load almost instantly - they take me a little while to load on my end.

Huh, you're right, we're both in Cincinnati! I've actually met two other SE4 players, Deslok and Geoschmo, who live in the Dayton area - maybe we should organize a SE4 get-together sometime!

**Comment by bpleshek** [comment]
Sounds good about the meetup sometime.  

Before I saw your repository I was starting to make notes of my own to redo(re-imagine) SE5 on Unity.   My first SE game was 5, though I have experience with many of the genre's predecessors.  I actually was going to import the data from the data files once and from then on make my own changes as needed(there were some things I wanted to change, some of which the Balance mod fixed).  But as I look at what you've done, I never thought of actually building it so that existing mods would work.

**Comment by ekolis** [comment]
Well, now I'm seeing this bug myself, running the game on .NET 5...

**Comment by ekolis** [comment]
To work around it, resize the window; then for some reason the names appear.

**Comment by ekolis** [comment]
I swear someone fixed this already...

**Comment by ekolis** [comment]
Yeah, I remember this being fixed last time I tried to start a game
